### PR TITLE
[ticket/15330] Twig function to know if a language string is defined

### PR DESCRIPTION
**Pull request by rubencm**
PHPBB3-15330

Checklist:

- [x] Correct branch: master for new features; 3.2.x, 3.1.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master / 3.2.x](https://area51.phpbb.com/docs/master/coding-guidelines.html), [3.1.x](https://area51.phpbb.com/docs/31x/coding-guidelines.html)
- [x] Commit follows commit message [format](https://wiki.phpbb.com/Git#Commit_Messages)

Tracker ticket (set the ticket ID to **your ticket ID**):

https://tracker.phpbb.com/browse/PHPBB3-15330

### CODE REVIEW

File: phpBB/phpbb/template/twig/extension.php [hunk 1]
```diff
@@ -182,4 +183,14 @@ function lang()
 
 		return call_user_func_array(array($this->language, 'lang'), $args);
 	}
+
+	/**
+	* Check if a language variable exist
```
Comment (Elsensee) [inline phpBB/phpbb/template/twig/extension.php]: *exists

I think, at least.

File: phpBB/phpbb/template/twig/extension.php [hunk 2]
```diff
@@ -182,4 +183,14 @@ function lang()
...
+	*
+	* @return bool
+	*/
+	public function lang_defined($key)
```
Comment (Elsensee) [inline phpBB/phpbb/template/twig/extension.php]: I'd rather call it `is_lang` but I guess this is fine too. Don't know what others are thinking about this, so just let it that way. Just wanted to share my opinion.

Comment (rubencm) [reply phpBB/phpbb/template/twig/extension.php]: I can change if it you want

Comment (Nicofuma) [reply phpBB/phpbb/template/twig/extension.php]: personally I don't really have a preference

Comment (Elsensee) [reply phpBB/phpbb/template/twig/extension.php]: @rubencm Then just leave it as it is. But change the exist to exists as stated in my previous comment. I'll merge it then.

**Pull request merged**